lib/CoreDisTools/.nuget/runtime.osx.10.10-x86.Microsoft.NETCore.CoreDisTools.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>runtime.osx.10.10-x86.Microsoft.NETCore.CoreDisTools</id>
+    <version>1.0.1-prerelease-00001</version>
+    <title>Microsoft.NETCore Instruction-wise Disassembler</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
+    <projectUrl>https://github.com/dotnet/coreclr</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Disassembly Tools for CoreCLR</description>
```

Comment:
Can we make this description string consistent across all the files (including the x64 ones)?